packages/segment/src/notifier-adapter.ts

```diff
@@ -0,0 +1,66 @@
+import { AnalyticsNotifierAdapter } from '@evolv-integrations/analytics-adapter';
+
+export class SegmentNotifierAdapter extends AnalyticsNotifierAdapter {
+    constructor(
+        public parametersToReadFromSegment: Record<string, any> = {},
+        public readonly maxWaitTime = 5000
+    ) {
+        super(maxWaitTime);
+    }
+
+    getAnalytics() {
+        // @ts-ignore
+        return window.analytics && window.analytics.track;
+    }
+
+    getHandler() {
+        return this.getAnalytics();
+    }
+
+    sendMetrics(type: string, event: any) {
+        let value: Record<string, any> = {
+            type
+        };
+
+        if (event.group_id) {
+            value = Object.assign(value, {
+                groupId: event.group_id,
+                ordinal: event.ordinal,
+                cid: event.cid
+            });
+        }
+        const isEvolvEvent = (name: string) => {
+            const eventsMap = Object.values(this.parametersToReadFromSegment);
+
+            if (Array.isArray(this.parametersToReadFromSegment[name])) {
+                this.parametersToReadFromSegment[name].forEach(({ event }: { event: string }) => {
+                    eventsMap.push(event);
+                });
+            } 
+
+            return eventsMap.indexOf(type) === -1;
+        };
+
+        if (type === 'confirmed' && event.group_id) {
```

Comment:
confirmed or contaminated

Reply:
This "Experiment Viewed" event should only be for the confirmation events. Segment's AB testing spec doesn't seem to have anything equivalent to a contamination event - so it is passed in the `else` condition below as `Evolv Event: contaminated`